common/src/main/java/org/opensearch/ml/common/model/TextEmbeddingModelConfig.java

```diff
@@ -97,12 +110,18 @@ public static TextEmbeddingModelConfig parse(XContentParser parser) throws IOExc
                 case MODEL_MAX_LENGTH_FIELD:
                     modelMaxLength = parser.intValue();
                     break;
+                case QUERY_PREFIX:
+                    queryPrefix = parser.text();
+                    break;
+                case PASSAGE_PREFIX:
+                    passagePrefix = parser.text();
+                    break;
```

Comment:
add test.

Reply:
This code is covered in common/src/test/java/org/opensearch/ml/common/model/TextEmbeddingModelConfigTests.java.